server/pyproject.toml

```diff
@@ -35,6 +35,9 @@ bnb = ["bitsandbytes"]
 grpcio-tools = "^1.51.1"
 pytest = "^7.3.0"
 
+[tool.pytest.ini_options]
```

Comment:
We don't have asyncio tests in the server.

Reply:
Ah! good catch